Gem/Code/Source/AutoGen/NetworkMatchComponent.AutoComponent.xml

```diff
@@ -13,6 +13,10 @@
     <!-- Round time is in seconds and is quantized to 2 bytes to reduce network traffic	-->
     <NetworkProperty Type="RoundTimeSec" Name="RoundTime" Init="RoundTimeSec{120}" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="true" IsPredictable="false" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="true" Description="The remaining time in the round in seconds" />
     <NetworkProperty Type="uint16_t" Name="RoundNumber" Init="1" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object" IsPublic="true" IsRewindable="true" IsPredictable="false" ExposeToEditor="false" ExposeToScript="false" GenerateEventBindings="true" Description="The current round number" />
+    <NetworkProperty Type="MatchResults" Name="Results" Init="MatchResults{}" ReplicateFrom="Authority" ReplicateTo="Client" Container="Object"
```

Comment:
I think you should set Container="Array" and Count="8" here. We aim to support 4 (or 8?) players.

Reply:
Right now a `MatchResults`* struct contains a string property _and_ an array of `PlayerState`s - we could nix that string prop and just send out the array, but I was thinking we'd likely extend `MatchResults` to include more top-level info (special achievements, or 2nd, 3rd place?) so an object still made sense. 

Is there a down side to this kind of organization which I'm not taking into account? Should the vector be extracted as its own property?

*now that I type it out, giving a singular struct a plural name isn't great. Should probably change the struct type to "MatchResultsSummary", or something like that.

Reply:
Discussed offline: will extract `PlayerState` property so it can be index-wise updated during game play, independent from the final results.